doc/reference/reference_lua/box_space/create_index.rst

```diff
@@ -98,8 +109,8 @@ On this page:
             |                     | faster, ``false`` -- an index size is reduced by half |                                  |                               |
             +---------------------+-------------------------------------------------------+----------------------------------+-------------------------------+
 
-            The options in the above chart are also applicable for
-            :doc:`/reference/reference_lua/box_index/alter`.
+        The options in the above chart are also applicable for
+        :doc:`/reference/reference_lua/box_index/alter`.
 
 
         **Note re storage engine:** vinyl has extra options which by default are
```

Comment:
"re" is not so popular :)

```suggestion
        **Note about storage engine:** vinyl has extra options which by default are
```

Reply:
We have this phrase declared here https://www.tarantool.io/en/doc/latest/book/box/engines/
So if you want to change the wording, let's do it in another issue.

Reply:
Hmm, okay.